source/PlayerInfo.cpp

```diff
@@ -149,6 +149,13 @@ void PlayerInfo::Load(const string &path)
 	// Make sure any previously loaded data is cleared.
 	Clear();
 
+	// A listing of missions and the ships where their cargo or passengers were when the game was saved.
+	// Missions and ships are referred to by string UUIDs.
+	// Any mission cargo or passengers that were in the player's system will not be recorded here,
+	// as that can be safely redistributed from the player's overall CargoHold to any ships in their system.
+	map<string, map<string, int>> missionCargoToDistribute;
```

Comment:
maybe short comment what the values in the map hold?

Reply:
Isn't that already described?
> A listing of missions and the ships where their cargo or passengers were when the game was saved.
> Missions and ships are referred to by string UUIDs.

That is, `map<mission UUID string, map<ship UUID string, cargo/passenger count>>`.

Reply:
if you think so